multinet/db.py

```diff
@@ -616,10 +615,10 @@ def count_text(filt: str) -> str:
 @lru_cache(maxsize=1)
 def uploads_database() -> StandardDatabase:
     """Return the database used for storing multipart upload collections."""
-    sysdb = db("_system")
+    sysdb = system_db(readonly=False)
     if not sysdb.has_database("uploads"):
         sysdb.create_database("uploads")
-    return db("uploads")
+    return db("uploads", readonly=False)
```

Comment:
The function should take a `readonly` parameter, like `workspace_mapping_collection()` does.